product_code/authentication_service/api/src/test/scala/de/upb/cs/uc4/authentication/AuthenticationUserSpec.scala

```diff
@@ -0,0 +1,38 @@
+package de.upb.cs.uc4.authentication

```

Comment:
We could think about moving every test to the exact same folder as the corresponding class for better overview.
(But not part of this PR).